fk/secretreceive.go

```diff
@@ -110,14 +111,16 @@ func secretReceive() exitCode {
 					}
 				}
 			}
-			if prompter.promptYesNo("Delete now?", "Y", nil) == true {
-				err := client.DeleteSecret(key.Fingerprint(), secret.UUID.String())
-				if err != nil {
-					log.Printf("failed to delete secret '%s': %v", secret.UUID, err)
-					printFailed("Error trying to delete secret:")
-					printFailed(err.Error())
-				}
+
+			err := client.DeleteSecret(key.Fingerprint(), secret.UUID.String())
+			if err != nil {
+				log.Printf("failed to delete secret '%s': %v", secret.UUID, err)
+				printFailed("Error when secret tried to self-destruct:")
+				printFailed(err.Error())
 			}
+
+			deletedSecret = true
```

Comment:
This is set even if there's an error deleting a secret, in which case "Secrets have self destructed!" would still be printed

Reply:
(Although I guess it has already printed the failure message at that point, so it's OK)